### DISCUSSION TITLE
help for run this site in github

### DISCUSSION THREAD

**Discussion by mohammadRezaeian** [dinhanhthi/dinhanhthi.com#34] [General]
hi i trying using your website for run in Github
but i can't find any file configuration for set URL
like old versions

**Comment by dinhanhthi** [comment]
Hi, sorry, the configuration is a bit complicated and I do not have time for a tutorial. By the way, I am using 11ty in the current version (5). You should know a little bit about 11ty to understand how this theme works and how to change it. I can not help you any further, sorry.
